news/2023/2023-07-25-flag-change-updates.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Country Flag Change Updates"
+date: 2023-07-26 13:00:00 +0000
+---
+
+Ever felt like you need a flag change? Look no further.
+
+![](/wiki/shared/news/2023-07-25-flag-change-updates/banner.jpg)
+
+Flag changes have been disabled from the osu! community and from osu! support since December 2021, and for the first time since then they have been reopened — with some minor adjustments.
```

Comment:
This sentence reads weird, saying its "disabled" makes it sound like its a button/toggle, and not a process you go through with support

```suggestion
The flag change process for the osu! community has been frozen since December 2021, and for the first time since then it has been resumed — with some minor adjustments.
```